main/electron.ts

```diff
@@ -135,8 +123,12 @@ export function handleSubmission(): void {
     let response: string | Error;
     try {
       // ToDo: move this to the frontend and pass a subset of the selected chapters across the IPC.
```

Comment:
This comment should be removed now that you have done it! Yeh!

Reply:
Done. I wasn't sure if this was what I had done or if it was referring to something else.